src/Orleans.Runtime/Silo/TestHooks/TestHooksSystemTarget.cs

```diff
@@ -63,18 +63,6 @@ public Task<bool> HasStreamProvider(string providerName)
             }
         }
 
-        public Task<bool> HasBoostraperProvider(string providerName)
```

Comment:
hmm should we support this in the legacy package too? If we support the legacy feature,   I think we should support the ability to test it too right?

Reply:
The reality is that these test hooks are mostly useful to test the internals of Orleans (aka: by us in Orleans.sln). Not really that important to provide this test hook.